protocol/daemons/pricefeed/client/client_integration_test.go

```diff
@@ -357,17 +382,25 @@ func (s *PriceDaemonIntegrationTestSuite) expectPricesWithTimeout(
 		if len(prices) != len(expectedPrices) {
 			continue
 		}
+
```

Comment:
I include a bugfix spelled out [in this PR](https://github.com/dydxprotocol/v4-chain/pull/657) for the sake of validating test cases. Happy to remove once that once gets approved!